packages/AlgernonTests.package/ALGTestKeyboard.class/instance/tearDown.st

```diff
@@ -1,5 +1,3 @@
 running
 tearDown
-
-	super tearDown.
-	toolWindow ifNotNil: [ toolWindow delete ]
```

Comment:
Here too, the line seems to be unnecessary.

Reply:
Yes, I think this is unnecessary. Please delete the method as that will be the same as calling `super tearDown`.

Reply:
Jep, please remove the method all together and remove the instance variable.